### DISCUSSION TITLE
Ikea E1743: messed up - how to reset bindings?

### DISCUSSION THREAD

**Discussion by ooohaaa** [Koenkk/zigbee2mqtt#9986] [General]
Hi all,
I'm using zigbee2mqtt in Home Assistant.
I'm new to this and was testing/trying some things the last days.
My current test setup consists of:

- Raspberry Pi 4 / 4GB
- Home Assistant / Zigbee2MQTT
- ConBee II Stick
- [Ikea Switch, E1743](https://www.zigbee2mqtt.io/devices/E1743.html#ikea-e1743)
- [Ikea Outlet, E1603/E1702/E1708](https://www.zigbee2mqtt.io/devices/E1603_E1702_E1708.html)

What I did so far:
I paird the switch and the outlet with zigbee2mqtt. Tried binding the switch to the outlet. Binding is listed at the Binding tab of z2m. But I couldn't switch on/off the outlet. Afterwards I saw this switch only supports binding to groups...

Now the problem is, that I can't delete the bindings. The binding tab looks likt this at the moment:
![grafik](https://user-images.githubusercontent.com/95161854/144666174-5ee21225-ed6a-4f48-b4dc-0e79d8f5a89c.png)

When I try to unbind the outlet, I get this Timeout error message:
`error Failed to unbind cluster 'genOnOff' from '0x847127fffe8f2c7a' to 'Steckdose 1' (Error: Unbind 0x847127fffe8f2c7a/1 genOnOff from '0x84fd27fffee0f0d8/1' failed (waiting for response TIMEOUT))
error Failed to unbind`

I also tried deleting the device. This was working earlier, but currently I need to use the **force remove** feature.
Afterwards I deleted the device from Mosquitto Broker. But after pairing the switch, all bindings (including the binding for the outlet) are here again...

**My questions:**
1. How to get back to a working state?
2. How to reset the device and/or bindings?
3. Where are the bindings stored?

Thanks in advance!

**Comment by 00joshi** [comment]
I solved the issue by

did you wake up the switch before trying to delete the binding?

1. delete the bindings / device
2. restart zigbee2mqtt
3. pair again and create new bindings
